#point of image

int32 h
int32 w
